iteration/iteration.go

```diff
@@ -253,6 +256,39 @@ func (m *GormIterationRepository) CanStart(ctx context.Context, i *Iteration) (b
 	return true, nil
 }
 
+func inTimeframe(startAt time.Time, endAt time.Time) bool {
+	if time.Now().UTC().After(startAt) && time.Now().UTC().Before(endAt) {
+		return true
+	} else {
+		return false
+	}
+}
+
+func (i *Iteration) IsActive() bool {
```

Comment:
Some unit tests for this function would be good.

Reply:
This commit contains unit tests 5698085